.env.local

```diff
@@ -0,0 +1,32 @@
+## NB only variables prefixed with NEXT_PUBLIC_SIMORGH_ are available on both the server and client
+NEXT_PUBLIC_SIMORGH_BASE_URL=http://localhost:3000
+NEXT_PUBLIC_SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN=http://localhost:3000
+NEXT_PUBLIC_SIMORGH_PUBLIC_STATIC_ASSETS_PATH=/static/
+NEXT_PUBLIC_SIMORGH_ATI_BASE_URL=https://logws1363.ati-host.net?
+NEXT_PUBLIC_SIMORGH_CONFIG_URL=https://config.test.api.bbci.co.uk/
+NEXT_PUBLIC_SIMORGH_INCLUDES_BASE_URL=https://www.test.bbc.com/ws/includes
+NEXT_PUBLIC_SIMORGH_INCLUDES_BASE_AMP_URL=https://news.test.files.bbci.co.uk
+NEXT_PUBLIC_SIMORGH_ICHEF_BASE_URL=https://ichef.bbci.co.uk
+LOG_DIR='log'
+LOG_LEVEL='debug'
+LOG_TO_CONSOLE=true
+NEXT_PUBLIC_SIMORGH_APP_ENV=local
+NEXT_PUBLIC_SIMORGH_CONFIG_CACHE_ITEMS=100
+NEXT_PUBLIC_SIMORGH_CONFIG_CACHE_MAX_AGE_SECONDS=60
+NEXT_PUBLIC_SIMORGH_CONFIG_TIMEOUT_SECONDS=5
+SERVER_KEEP_ALIVE_TIMEOUT=61000
+SERVER_HEADERS_TIMEOUT=65000
+NEXT_PUBLIC_SIMORGH_MOST_READ_CDN_URL=http://localhost:3000
+
+## CW Custom Metrics Configuration
+AWS_EMF_SERVICE_NAME=Simorgh
+AWS_EMF_LOG_GROUP_NAME=SimorghServer
+AWS_EMF_ENVIRONMENT=Local
+
+## WebVitals Reporting
+NEXT_PUBLIC_SIMORGH_WEBVITALS_REPORTING_ENDPOINT=https://europe-west1-bbc-otg-traf-mgr-bq-dev-4105.cloudfunctions.net/report-endpoint
+NEXT_PUBLIC_SIMORGH_WEBVITALS_DEFAULT_SAMPLING_RATE=100
+
+NEXT_PUBLIC_SIMORGH_CSP_REPORTING_ENDPOINT=https://ws.bbc-reporting-api.app/report-endpoint
+
+NEXT_PUBLIC_SIMORGH_OPTIMIZELY_SDK_KEY=LptPKDnHyAFu9V12s5xCz
```

Comment:
All of these env files moved from an `envConfig` folder, got renamed to work with Next.js's dotenv convention, and all variables used on the client got prefixed with `NEXT_PUBLIC_`.